packages/components/src/components/Collapsible/index.tsx

```diff
@@ -82,12 +81,12 @@ export const Collapsible: React.FC<ICollapsibleProps> = ({
   const toggle = () => setOpen(!open);
 
   return (
-    <Element as="section" {...props}>
+    <Section {...props}>
```

Comment:
I'm curious, how would it be to write the styles in place - 

```jsx
<Element as="section" css={{fontSize: 3}}>
```

vs 

```js
const Section = styled(Element).attrs({ as: 'section' })(
  css({
    fontSize: 3,
  })
);
```

Reply:
I perfer the top one but I think we should use the bottom one as it's more js like and I know drake also perfers that one